### ISSUE TITLE
Enable Share Dialog Window - when checked, prompts a login

### ISSUE THREAD

**Issue by Nataliya1** [Esri/crowdsource-reporter#475]
We have a publicly shared Citizen Reporter app, and we have been using it for a couple years. Recently, we made a slight adjustment to the app by enabling the share dialog window: Under Options >  **"Enable the share dialog window"**. Since then we got multiple reports from the Citizens that when they click on the 'Proceed as a Guest' button for the Citizen Reporter they are taken to the login screen and are prompted to login to the app. I was able to replicated the issue with our Prod and Dev Sites.

**Comment by aparveen** [comment]
@skrishn could you please take a look at this issue when you have a moment?

**Comment by skrishn** [comment]
Hi @Nataliya1 : I am unable to reproduce this issue. All your layer/map/app/group shared publicly? Can you share your app please

**Comment by Nataliya1** [comment]
Here is a link to my Dev site: https://omaha.maps.arcgis.com/apps/CrowdsourceReporter/index.html?appid=9dce422c735f460db4f05282ca6cd445 Yes, everything is shared publicly.

**Comment by skrishn** [comment]
Thanks @Nataliya1, I was able to repro it with your app with both Guest and Twitter login in the morning but not anymore. Has the issue been fixed at your end? Did you make any changes?

**Comment by Nataliya1** [comment]
@skrishn no, nothing has been fixed, or changed. But it seems like if you have logged in to AGO in your browser earlier, the credentials will be stored in cache, and the issue will not appear any more. Unless you clear your browser cache, or try different browser were you haven't logged in to the AGO in a while.

**Comment by skrishn** [comment]
@ashishchoure : Could you take a look at this issue. Thank you!


cc @aparveen

**Comment by ashishchoure** [comment]
@skrishn @aparveen 

- We are not able to reproduce the issue at our end.
- We tried with multiple browsers but the application was working as expected.
- Here is the small GIF, please let us know if we are missing anything from our side.

![CSR_475_1](https://user-images.githubusercontent.com/5525972/94512916-efd4ee00-023a-11eb-9a1c-57da1f2e24bb.gif)

**Comment by aparveen** [comment]
@Nataliya1 Since we could not repro the issue, I would suggest that you contact our technical support and log in this issue, they have the capability to look into your organization to figure out what is going on.

**Comment by Nataliya1** [comment]
@aparveen I opened a case. Thank you for looking into this.

**Comment by skrishn** [comment]
@Nataliya1 :  Just as an FYI,  this may be the cause for prompting issues in your apps (if you havent solved it already). It would happen if you're using proxy services for routing, geocoding, geoenrichment, living atlas. 
https://developers.google.com/web/updates/2020/07/referrer-policy-new-chrome-default mig

**Comment by Nataliya1** [comment]
@skrishn I was going to look into this, but it seems like both of my Prod and Dev apps are working fine this morning, no login screen. And I did absolutely nothing. I tried loading apps multiple times, in Chrome and Firefox, using incognito window and no more login screen!
